core/src/database/postgres/generate.rs

```diff
@@ -151,7 +151,16 @@ fn generate_internal_event_table_sql(
             )
         }).collect::<Vec<_>>().join("\n");
 
-        format!("{}\n{}", create_table_query, insert_queries)
+        let create_latest_block_query = r#"CREATE TABLE IF NOT EXISTS rindexer_internal.latest_block ("network" TEXT PRIMARY KEY, "block" NUMERIC);"#.to_string();
```

Comment:
Tracking latest seen block in DB for external / more flexible progress update reporting